index.js

```diff
@@ -55,25 +53,39 @@ function main (options, done) {
     if (err) {
       throw new Error(err)
     }
-
     console.log(`Authenticated as ${user.username}`.blue)
-    client.createPost({
-      userId: user.id,
-      title,
-      tags,
-      content,
-      canonicalUrl,
-      contentFormat: 'markdown',
-      publishStatus: 'draft'
-    }, (err, post) => {
+    const options={
+        userId: user.id,
+        title,
+        tags,
+        content,
+        canonicalUrl,
+        contentFormat: 'markdown',
+        publishStatus: 'draft'
+    };
+    const successMsg=`Draft post "${title}" published to Medium.com`.green;
+    publication?client.getPublicationsForUser({userId:user.id}, (err, publications)=>{
```

Comment:
Using a ternary here is a bit confusing. Could you break it into an if block for readability?